# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/moz.build
# Commit: 28bca5c5b8b4
# Full Hash: 28bca5c5b8b4d5a4f8e6ecaf621738ad955c3121
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-09 15:16:49
# Regressor Bug: 1641737
# File Overlap Count: 2
# Description:
#   Bug 1641737 - P2. Use MozPromise with DocumentLoadListener and remove cycle. r=mattwoodrow.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78179
# ==============================================================================

diff -r 0296d0f6c1d1 -r 28bca5c5b8b4 netwerk/ipc/moz.build
--- a/netwerk/ipc/moz.build	Tue Jun 09 07:23:29 2020 +0000
+++ b/netwerk/ipc/moz.build	Tue Jun 09 10:11:11 2020 +0000
@@ -5,7 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 EXPORTS.mozilla.net += [
-    'ADocumentChannelBridge.h',
     'ChannelEventQueue.h',
     'DocumentChannel.h',
     'DocumentChannelChild.h',
